misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -873,13 +875,13 @@ public AssignmentStatementNode transform(AssignmentStatementNode assignmentState
         Node varRef = formatNode(assignmentStatementNode.varRef(), 1, 0);
         Token equalsToken = formatToken(assignmentStatementNode.equalsToken(), 1, 0);
 
-        boolean previousinLineAnnotation = env.inLineAnnotation;
+        boolean previousInLineAnnotation = env.inLineAnnotation;
         setInLineAnnotation(true);
 
         ExpressionNode expression = formatNode(assignmentStatementNode.expression(), 0, 0);
         Token semicolonToken = formatToken(assignmentStatementNode.semicolonToken(), env.trailingWS, env.trailingNL);
 
-        setInLineAnnotation(previousinLineAnnotation);
+        setInLineAnnotation(previousInLineAnnotation);
```

Comment:
```suggestion
        setInLineAnnotation(previousInlineAnnotation);
```

Reply:
Since inline is a one word, all places where inline is treated as two words were changed to preserve consistency